src/main/java/com/getindata/connectors/http/internal/table/sink/HttpDynamicSink.java

```diff
@@ -180,6 +190,18 @@ public HttpDynamicTableSinkBuilder setEncodingFormat(
             return this;
         }
 
+        /**
+         * @param httpSinkPostRequestCallback the {@link HttpSinkPostRequestCallback} implementation
+         *                                    for processing of requests and responses
+         * @return {@link HttpDynamicTableSinkBuilder} itself
+         */
+        public HttpDynamicTableSinkBuilder setHttpSinkPostRequestCallback(
+            HttpSinkPostRequestCallback httpSinkPostRequestCallback
+        ) {
```

Comment:
please move `) {` to line 199

Reply:
IMO its matter of checstyle rule so spotless will also have to have that rule we are now missing. 

We already have a quality gate that will fail build if checkstyle is violated. 
Also already now you can reformat code by executing Intellij "ctrl + alt + l" command that should fix your class if you have proper checstyle config as described in "https://github.com/getindata/flink-http-connector/tree/main/dev"

Personally I would avoid any auto tools that change code without developer knowledge. 
I would argue that it should be correct at the source and there should be gates/checks that will not allow for merge/build if something is off.